### PR TITLE
Wire referenceRelation to store

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: T272699

### CODE REVIEW

File: src/components/QueryCondition.vue [hunk 1]
```diff
@@ -41,7 +41,6 @@
 				<ReferenceRelationDropDown
 					class="query-condition__references"
 					v-model="selectedReferenceRelation"
-					:disabled="true"
```
Comment (micgro42) [inline src/components/QueryCondition.vue]: This doesn't actually work yet, right? In the sense of producing the correct SPARQL? Then I would say let's keep it disabled for now until the user can actually use it.

**Review by micgro42 — CHANGES_REQUESTED** [review]
I think that enabling that dropdown in this PR might be a bit premature. But maybe I'm missing something?

File: src/components/QueryCondition.vue (hunk 1 above)
Comment (Ladsgroup) [reply src/components/QueryCondition.vue]: Yeah, I enabled it to test it but forgot to disable it back again.

**Review by micgro42 — APPROVED** [review]
\o/

**Pull request merged**